components/dialog/dialog-styles.js

```diff
@@ -0,0 +1,74 @@
+import { css } from 'lit-element/lit-element.js';
+
+export const dialogStyles = css`
+
+	:host {
+		display: none;
+	}
+
+	:host([opened]), :host([_state="showing"]), :host([_state="hiding"]) {
+		display: block;
+	}
+
+	.d2l-dialog {
+		background-color: white;
+		border: 1px solid var(--d2l-color-mica);
+		border-radius: 8px;
+		box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+		overflow: hidden;
+		top: 100px;
+		transform: translateY(-50px);
+		transition: transform 200ms ease-in;
+	}
+
+	div.d2l-dialog {
```

Comment:
Guessing this is for polyfilled vs. `dialog.d2l-dialog` is not?

Reply:
Correct.